ffi/diplomat/src/fixed_decimal.rs

```diff
@@ -108,6 +108,25 @@ pub mod ffi {
             self.0.negate()
         }
 
+        /// Zero-pad the [`ICU4XFixedDecimal`] on the left to a particular number of integer digits
+        /// See [the Rust docs](https://unicode-org.github.io/icu4x-docs/doc/fixed_decimal/decimal/struct.FixedDecimal.html#method.pad_left) for more information.
+        pub fn pad_left(&mut self, digits: u16) {
+            self.0.pad_left(digits)
+        }
+
+        /// Truncate the [`ICU4XFixedDecimal`] on the left to a particular magnitude, deleting digits if necessary. This is useful for, e.g. abbreviating years
+        /// ("2022" -> "22")
+        /// See [the Rust docs](https://unicode-org.github.io/icu4x-docs/doc/fixed_decimal/decimal/struct.FixedDecimal.html#method.pad_left) for more information.
```

Comment:
Nit: The link goes to the wrong function

Reply:
argh, fixed